Installers/Linux/RUN/postinstall.sh

```diff
@@ -88,6 +84,18 @@ case "$choice2" in
 	sudo -H -u $SUDO_USER bash -c 'mdtool setup install $DIR/Main/MonoDevelop.MonoGame.mpack'
 esac
 
+#MonoGame.Content.Builder.targets
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame
+fi
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame/v3.0 ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame/v3.0
+fi
+
+sudo cp $DIR/Main/MonoGame.Content.Builder.targets tmp/usr/lib/mono/xbuild/MonoGame/v3.0/
```

Comment:
@cra0zy 
Should this be tmp/usr/lib/..... ?

Also with the tooling the XS/VS Templates and targets will expect the MGCB.exe (and its dependencies) to be in 
```
/usr/lib/mono/xbuild/MonoGame/v3.0/Tools/
```
and the Assemblies to be in 
```
/usr/lib/mono/xbuild/MonoGame/v3.0/Assemblies/<Platform>
```
On Mac we actually install into a versioned directory and symlink a v3.0 folder, that means we can still know which actual version we have installed. Not sure if we want to do that on linux too.